api_app/views.py

```diff
@@ -148,54 +209,63 @@ def ask_analysis_availability(request):
     )
 
     serializer = JobAvailabilitySerializer(
-        data=data_received, context={"request": request}
+        data=data_received, context={"request": request}, many=True
```

Comment:
This endpoints now expect a `list of jobs`. I can see stuff breaking with this change (pyintelowl).
Again, i would prefer that we check the input type and manage it internally the function, without  having breaking changes in the project.